managing-os/fedora-template-upgrade-21.md

```diff
@@ -54,11 +54,17 @@ template based on the standard Fedora 21 template.
         [user@fedora-23 ~]$ sudo yum clean all
         [user@fedora-23 ~]$ sudo yum --releasever=23 --enablerepo=qubes*current-testing distro-sync
 
-    (Shut down TemplateVM via Qubes VM Manager; may need to be killed.)
-
+ 3. Shutdown the new TemplateVM via dom0 command line or Qubes VM Manager; may need to be killed.)
+
+        [user@dom0 ~]$ qvm-shutdown fedora-23
+    
+    or
+    
+        [user@dom0 ~]$ qvm-shutdown --force fedora-23
```

Comment:
Did you encountered any case when `--force` is needed? I think it would be better to use `--wait`, which would wait for VM shutdown and after 60sec timeout will kill it.